references/depth/stereo/transforms.py

```diff
@@ -0,0 +1,572 @@
+import random
+from typing import List, Sequence, Tuple, Union
+
+import numpy as np
+import PIL.Image
+import torch
+import torchvision.transforms as T
+import torchvision.transforms.functional as F
+from torch import Tensor
+
+T_FLOW = Union[Tensor, np.ndarray, None]
+T_MASK = Union[Tensor, np.ndarray, None]
+T_STEREO_TENSOR = Tuple[Tensor, Tensor]
```

Comment:
My main curiosity is wether or not we should wrap this in a custom data type that can handle the transform operations in order to skip littering the code with IF-ELSE logic blocks when dealing with datasets that miss that specific piece of information in one of the pairs.